src/plugins/vis_type_timeseries/public/application/components/lib/get_supported_fields_by_metric_type.js

```diff
@@ -25,6 +25,7 @@ export function getSupportedFieldsByMetricType(type) {
     case METRIC_TYPES.CARDINALITY:
       return Object.values(KBN_FIELD_TYPES).filter((t) => t !== KBN_FIELD_TYPES.HISTOGRAM);
     case METRIC_TYPES.VALUE_COUNT:
+      return [KBN_FIELD_TYPES.NUMBER, KBN_FIELD_TYPES.HISTOGRAM, KBN_FIELD_TYPES.STRING];
```

Comment:
It seems like value count works for all field types - I changed this to `return KBN_FIELD_TYPES` and got nice looking charts as well:
<img width="1709" alt="Screenshot 2020-10-05 at 15 28 11" src="https://user-images.githubusercontent.com/1508364/95085396-75670b00-071f-11eb-9b5f-9a314e1a9a77.png">

Reply:
No if it supports them all then I will change it to support them all 🙂 Thanx for that